addons/interact_menu/initSettings.sqf

```diff
@@ -7,3 +7,63 @@
     false,
     {GVAR(selectorColorHex) = _this call BIS_fnc_colorRGBtoHTML} // Stored in Hex to avoid constant conversion
 ] call CBA_settings_fnc_init;
+
+[
+    QGVAR(alwaysUseCursorInteraction),
+    "CHECKBOX",
+    localize LSTRING(AlwaysUseCursorInteraction),
+    [format ["ACE %1", localize LSTRING(Category_InteractionMenu)], localize LSTRING(Category_InteractionMenu)],
+    false,
+    false,
+    {GVAR(alwaysUseCursorInteraction) = _this}
```

Comment:
such onChange code is useless, the var is set automatically. Here and below.

Reply:
Cool, I'll take it out.